### PR TITLE
fix(docs): change style for cards in overview of component.

### PR DESCRIPTION
**Pull request by HarshJainZeusInn**
### What does this implement/fix? Explain your changes.
 - In normal state opacity of cards in overview of component is 0.6 and on hover the opacity becomes 1 .
...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: docs/src/pages/components/overview/tabs/overview.css [hunk 1]
```diff
@@ -28,3 +27,7 @@
 .card-link[disabled], .card-link[disabled] * {
   pointer-events: none;
 }
+
+/* .cardOpacity {
+  background-color: red;
+} */
```
Comment (anuradha9712) [inline docs/src/pages/components/overview/tabs/overview.css]: Please remove this commented code

**Comment by anuradha9712** [comment]
Please change the commit message

File: docs/src/pages/components/overview/tabs/overview.css (hunk 1 above)
Comment (HarshJainZeusInn) [reply docs/src/pages/components/overview/tabs/overview.css]: > Please remove this commented code

done

**Comment by HarshJainZeusInn** [comment]
> Please change the commit message

done

**Review by anuradha9712 — DISMISSED** [review]

**Review by anuradha9712 — APPROVED** [review]

**Review by veekays — APPROVED** [review]

**Pull request merged**